### PR TITLE
Dnn pose estimation

### PR DESCRIPTION
**Pull request by berak**
replaced the lightweight_pose_estimation.onnx model with a correct version, updated test data

relates https://github.com/opencv/opencv/issues/16204

### CODE REVIEW

File: testdata/dnn/download_models.py [hunk 1]
```diff
@@ -788,8 +788,8 @@ def save(self, r):
         filename='onnx/models/facial_keypoints.onnx'),
     Model(
         name='LightWeight Human Pose Estimation (ONNX)', # https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch
-        url='https://drive.google.com/uc?export=dowload&id=1T2Kq01WXzPMrQdnEOUEiVBhwouW8Pka5',
-        sha='20370164b8c43aa14625c0be4e31f62b9b397ac4',
+        url='https://drive.google.com/uc?export=dowload&id=1--Ij_gIzCeNA488u5TA4FqWMMdxBqOji',
+        sha='5960f7aef233d75f8f4020be1fd911b2d93fbffc',
         filename='onnx/models/lightweight_pose_estimation.onnx'),
```
Comment (alalek) [inline testdata/dnn/download_models.py]: Please use different filename for modified/replaced files (add suffix like `_201912`).
(versioning is not properly handled for this script)

**Comment by berak** [comment]
ah, i see, thanks for looking into it !

**Comment by dkurt** [comment]
Maybe it's better to have keypoints from the first 19 heatmaps in the test instead of numpy blob?

**Comment by berak** [comment]
@dkurt , good idea, less work to change something later.

same for the facial keypoints, i guess ?

**Pull request merged**